pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -341,6 +341,33 @@ self: super:
           "/rdrand/rngAddEntropy/"
         ]);
     }) super.botan-low;
+
+    # 2025-11-07: Broken since 2015 - https://github.com/NixOS/nixpkgs/pull/459465
+    hamid = markBroken super.hamid;
+    # 2025-11-07: Broken since 2016 - https://github.com/NixOS/nixpkgs/pull/459465
+    huckleberry = markBroken super.huckleberry;
+    log = markBroken super.log;
+    # 2025-11-07: Broken since 2017 - https://github.com/NixOS/nixpkgs/pull/459465
+    gtk-traymanager = markBroken super.gtk-traymanager;
+    pipes-zlib = markBroken super.pipes-zlib;
+    pthread = markBroken super.pthread;
+    # 2025-11-07: Broken since 2018 - https://github.com/NixOS/nixpkgs/pull/459465
+    system-linux-proc = markBroken super.system-linux-proc;
+    # 2025-11-07: Broken since 2021 - https://github.com/NixOS/nixpkgs/pull/459465
+    gerrit = markBroken super.gerrit;
+    # 2025-11-07: Broken since 2023 - https://github.com/NixOS/nixpkgs/pull/459465
+    copilot-c99 = markBroken super.copilot-c99;
+    error-codes = markBroken super.error-codes;
+    epub-metadata = markBroken super.epub-metadata;
+    HsHTSLib = markBroken super.HsHTSLib;
+    zlib-bytes = markBroken super.zlib-bytes;
+    # 2025-11-07: Broken since 2024 - https://github.com/NixOS/nixpkgs/pull/459465
+    bytestring-encoding = markBroken super.bytestring-encoding;
+    duckdb-haskell = markBroken super.duckdb-haskell;
+    freetype2 = markBroken super.freetype2;
```

Comment:
You can mark this as broken on aarch64-linux as well, https://github.com/NixOS/nixpkgs/pull/459465/files#r2506985396, https://github.com/dagit/freetype2/issues/10

Reply:
I see that the original issue is fixed, so I assume if this is not fixed already for aarch64-linux it will be fixed soon.